src/components/HomeHighlight/HighlightItem.css

```
@@ -0,0 +1,87 @@
+
+.highlight-Item{
```

Aplicar como padrão do projeto
```suggestion
.highlight-Item {
```